js/src/utils/createMessageForMultipleErrors.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { __, sprintf, _n } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import concatenateListOfWords from '.~/utils/concatenateListOfWords';
+
+/**
+ * Creates a message for a list of error messages. For example for a list of error messages: createMessageForMultipleErrors(['Target Audience', 'Shipping Rates']) will create the following message:
+ * 'There are errors in the following actions: Target Audience and Shipping Rates. Please try again later.'
+ *
+ * @param {Array<string>} errorMessages Error messages to be joined.
+ * @param {boolean} [isPartiallySuccessful='true'] Whether some actions were successful.
+ * @return {string|null} - Return the error message for a list of errors otherwise null if there are no error messages
+ */
+export default function createMessageForMultipleErrors(
+	errorMessages,
+	isPartiallySuccessful = true
+) {
+	if ( errorMessages.length ) {
+		const listErrors = concatenateListOfWords( errorMessages );
```

Comment:
:construction: We are not providing i18nized separators here, so when using Polish translation I will see:
> Podane akcje niepowiodły się: Koszt Dostawy **and** Cena Dostawy.  Spróbuj ponownie.

instead of 
> Podane akcje niepowiodły się: Koszt Dostawy **i** Cena Dostawy. Spróbuj ponownie.

Reply:
Thanks for pointing out this. TIL

The suggestion in https://github.com/woocommerce/google-listings-and-ads/pull/1602#discussion_r929526634 that includes `'and'` text in the i18n function could be considered.

Reply:
Fixed in https://github.com/woocommerce/google-listings-and-ads/pull/1602/commits/340a524876435634fea495ca977d082f74f67ea3.